governance.md

```diff
@@ -0,0 +1,57 @@
+
+The software developed by The Project is released under the MIT (or similar) open source license, developed openly and hosted in public GitHub repositories under the [Open Journals GitHub organization](https://github.com/openjournals). Examples of Project Software include the [JOSS web application](https://github.com/openjournals/joss), the [Whedon RubyGem](https://github.com/openjournals/whedon), the [Whedon bot](https://github.com/openjournals/whedon-api) and more.
+
+The services run by The Project consist of public websites and web-services that are hosted under the `theoj.org` and `briefideas.org` domain and include the Journal of Open Source Software (http://joss.theoj.org), the Journal of Open Source Education (http://jose.theoj.org), the Journal of Brief Ideas (http://beta.briefideas.org), and the Open Journal of Astrophysics (http://astro.theoj.org).
+
+The Project is formally affiliated with the 501c3 NumFOCUS Foundation (http://numfocus.org), which serves as its fiscal sponsor, may hold project trademarks and other intellectual property, helps manage project donations and acts as a parent legal entity. In addition, one of our journals, the Journal of Open Source Software, is formally affiliated with the Open Source Initiative (https://opensource.org/affiliates/list). NumFOCUS is the only legal entity that has a formal relationship with The Project.
+
+## Governance
+
+This section describes the governance and leadership model of The Project.
+
+The foundations of Project governance are:
+
+- Openness & Transparency
+- Active Contribution
+- Institutional Neutrality
+
+The Project leadership consists of a BDFL (Arfon Smith) and Steering Council.
+
+### BDFL
+
+The Project will have a BDFL (Benevolent Dictator for Life), who is currently Arfon Smith. As Dictator, the BDFL has the authority to make all final decisions for The Project. As Benevolent, the BDFL, in practice chooses to defer that authority to the consensus of the community discussion channels and the Steering Council (see below). It is expected, and in the past has been the case, that the BDFL will only rarely assert his/her final authority. Because rarely used, we refer to BDFL’s final authority as a “special” or “overriding” vote. When it does occur, the BDFL override typically happens in situations where there is a deadlock in the Steering Council or if the Steering Council asks the BDFL to make a decision on a specific matter. To ensure the benevolence of the BDFL, The Project encourages others to fork the project if they disagree with the overall direction the BDFL is taking. The BDFL is chair of the Steering Council (see below) and may delegate his/her authority on a particular decision or set of decisions to any other Council member at his/her discretion.
```

Comment:
The Project will have a BDFL (Benevolent Dictator for Life), who is currently Arfon Smith. As Dictator, the BDFL has the authority to make all final decisions for The Project. The BDFL, being benevolent, in practice chooses to defer that authority to the consensus of the community discussion channels and the Steering Council (see below). It is expected, and in the past has been the case, that the BDFL will only rarely assert his/her final authority. Because rarely used, we refer to BDFL’s final authority as a “special” or “overriding” vote. When it does occur, the BDFL override typically happens in situations where there is a deadlock in the Steering Council or if the Steering Council asks the BDFL to make a decision on a specific matter. To ensure the benevolence of the BDFL, The Project encourages others to fork the project if they disagree with the overall direction the BDFL is taking. The BDFL is chair of the Steering Council (see below) and may delegate his/her authority on a particular decision or set of decisions to any other Council member at his/her discretion.